zipline/assets/assets.py

```diff
@@ -264,25 +264,39 @@ def populate_cache(self):
         Populates the asset cache with all values in the assets
         collection.
         """
-
         # Wipe caches before repopulating
         self.cache = {}
```

Comment:
@ssanderson I think you need to rebase against master.
The `populate_cache` method has been removed.